components/CreateRecipeForm.js

```diff
@@ -99,12 +102,12 @@ function CreateRecipeForm(props) {
         );
     };
 
-    const addImage = uri => {
-        setRecipe(oldRecipe => ({
-            ...oldRecipe,
-            img: uri,
-        }));
-    };
+    // const addImage = uri => {
+    //     setRecipe(oldRecipe => ({
+    //         ...oldRecipe,
+    //         img: uri,
+    //     }));
+    // };
```

Comment:
Are you no longer using this function? If so, please delete to eliminate "zombie code".

Reply:
Yep, I'll definitely get rid of it.